### ISSUE TITLE
Latest Windows binaries 

### ISSUE THREAD

**Issue by Trid-collab** [JohnnyMorganz/StyLua#406]
There seems to some problem with the latest windows binaries. After downloading while extracting it says invalid zip or no files inside it

**Comment by JohnnyMorganz** [comment]
Just tested this out locally, and it seems to download, extract and work fine on my end.
Could your download of the zip possibly be corrupted?

**Comment by Trid-collab** [comment]
Let me try again. I tried downloading with curl as well with the same results

**Comment by Trid-collab** [comment]
Yes, it works. You were right maybe somehow my zip download was getting corrupted
